### PR TITLE
Show multiple network interfaces in UI

### LINKED ISSUE #686: Support multiple network interfaces
Update the API and UI to support and show data from multiple network interfaces.

I think we had an issue for it, but couldn't find one.

v2.0 API supports multiple interfaces, but doesn't yet fill in the data. Taking this up, as I started on it earlier.

### PR DESCRIPTION
**Pull request by vmarmol**
Fixes #686

/cc @rjnagal

### CODE REVIEW

File: pages/containers_html.go [hunk 1]
```diff
@@ -192,6 +192,16 @@ const containersHtmlTemplate = `
 	  <div class="panel-heading">
             <h3 class="panel-title">Network</h3>
 	  </div>
+          <div class="panel-body">
+	    <div class="dropdown">
+              <button class="btn btn-default dropdown-toggle" type="button" id="network-selection-dropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (rjnagal) [inline pages/containers_html.go]: keep expanded by default?

Comment (vmarmol) [inline pages/containers_html.go]: I'm okay with that, but won't it block part of the graph if so?

Comment (rjnagal) [inline pages/containers_html.go]: nvmd, I see what you did there

**Comment by rjnagal** [comment]
LGTM

this is pretty cool and clean! Thanks @vmarmol

**Comment by AJNOURI** [comment]
Hi guys, I am running an openvswitch container with 16 interfaces and I need to monitor all interfaces. Only 6 are shown in the dropdown menu, is there a reason behind the limitation?

**Pull request merged**